lib/install/validate/validator.go

```diff
@@ -163,6 +163,15 @@ func NewValidator(ctx context.Context, input *data.Data) (*Validator, error) {
 		op.Debugf("new validator Session.Populate: %s", err)
 	}
 
+	if v.Session.VMFolder == nil {
```

Comment:
I think this may cause issues when doing a `vic-machine ls` with multiple datacenters.

I suggest delaying any further work on this until @matthewavery has delivered the inventory folder support (#773) and then revisting it.

Reply:
makes sense. I will then leave the PR as it is for now, and move the ticket back to To-Do. Thanks!